src/models/predictors/mlp_predictor.py

```
@@ -47,21 +47,21 @@ def load_model_from_mlflow(self) -> None:
+            f"Expected features: {len(self.model.feature_names) if hasattr(self.model, 'feature_names') and self.model.feature_names else 'Unknown'}"
         )
-        logger.info(f"   Input size: {self.model.config.get('input_size', 'Unknown')}")
+        logger.info(f"Input size: {self.model.config.get('input_size', 'Unknown')}")
 
         # Check if scaler was loaded
         if hasattr(self.model, "scaler") and self.model.scaler is not None:
-            logger.info("   ✅ Scaler loaded from MLflow artifacts")
+            logger.info("Scaler loaded from MLflow artifacts")
         else:
-            logger.info("   ℹ️ No scaler found - will use raw features")
+            logger.info("ℹ️ No scaler found - will use raw features")
```

Ruff (0.14.4)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)